### DISCUSSION TITLE
switch between the album version and the official video

### DISCUSSION THREAD

**Discussion by grumi123** [pear-devs/pear-desktop#777] [Ideas]
sorry im using a transaltor mi english is bad 

Just like the premium feature, it would be nice to have a feature like that, sometimes there are some music videos with intros or outros that can make the music listening experience worse.

there is a solution, play the album from a playlist but it would be nice to include a feature like that.

I am sorry if you did not understand me to clarify I am referring to this  [https://support.google.com/youtubemusic/answer/6313574?hl=en&co=GENIE.Platform%3DAndroid](url)

**Comment by Araxeus** [comment]
There are 2 plugins that could help you:
* `video-toggle` with 'native' mode
* `sponsorblock` which skips non music parts in videos
